opsdroid/connector/slack/__init__.py

```diff
@@ -25,8 +26,11 @@ def __init__(self, config, opsdroid=None):
         self.icon_emoji = config.get("icon-emoji", ":robot_face:")
         self.token = config["api-token"]
         self.timeout = config.get("connect-timeout", 10)
-        self.slack = slack.WebClient(token=self.token, run_async=True)
-        self.slack_rtm = slack.RTMClient(token=self.token, run_async=True)
+        ssl_context = ssl.create_default_context()
+        ssl_context.check_hostname = False
+        ssl_context.verify_mode = ssl.CERT_NONE
```

Comment:
I don't feel comfortable disabling SSL verification here.

Reply:
This could be a temporary fix until the issue/documentation with the underlying library is resolved.

Reply:
Heh there is no such thing as temporary. 

But seriously it's not a good idea to introduce a security flaw just to work around a bug. This is the kind of thing that needs fixing properly.

The other option could be to pin to the older version (and work out why the `ibm-watson` package isn't happy). @FabioRosado could you help out with that?

Reply:
Got it. Will try to fix it without disabling SSL verification.

Reply:
Used `certifi` to fix the ssl issue.

Reply:
I had a look at the ibm-watson  library and they set `websocket-client==0.48.0` so that's the reason why its failing :/

Reply:
Sure no worries. I'm happy with `certifi` as a solution.